msg/viconEstimator.msg

```diff
@@ -1,13 +1,22 @@
 Header header
 
-geometry_msgs/Vector3 		pos_measured 	# the measured body position
-geometry_msgs/Vector3 		pos_old 			# the old body position
-geometry_msgs/Vector3 		vel_old  			# the old body velocity
-geometry_msgs/Vector3 		pos_est				# the posteriori body position
-geometry_msgs/Vector3 		vel_est				# the posteriori body velocity
-
-geometry_msgs/Quaternion 	quat_measured # the measured body orientation
-geometry_msgs/Quaternion 	quat_old 			# the old body orientation
-geometry_msgs/Vector3 		omega_old  		# the old body rate
-geometry_msgs/Quaternion 	quat_est			# the posteriori body orientation
-geometry_msgs/Vector3 		omega_est			# the posteriori body rate
\ No newline at end of file
+geometry_msgs/Vector3     pos_measured           # the measured body position
```

Comment:
Yes it is. I have had thoughts about removing it. But I've been thankful for not removing it a couple of times now. At some point though when things are stable it should be removed.